libs/community/langchain_community/llms/gpt4all.py

```diff
@@ -211,3 +214,40 @@ def _call(
         if stop is not None:
             text = enforce_stop_tokens(text, stop)
         return text
+
+    async def _acall(
```

Comment:
doesn't LLM._acall have a default implementation that does the same thing https://github.com/langchain-ai/langchain/blob/304f3f5fc1517adf54f9d022f13e8209749b00c1/libs/core/langchain_core/language_models/llms.py#L1109

cc @nfcampos

Reply:
Okey, that is true. It has that method implemented. I tested it also and it works the same as this one.
My PR is linked to this issue https://github.com/langchain-ai/langchain/issues/5210, if the issue is solved already, then close both the issue and this PR.

Reply:
Was the current implementation for GPT4All tested with websockets for example (to test the asynchronous behavior works as intended?) (I will double check this ASAP)